azure-spring-boot/src/main/java/com/microsoft/azure/keyvault/spring/KeyVaultPropertySource.java

```diff
@@ -19,7 +19,7 @@ public KeyVaultPropertySource(KeyVaultOperation operation) {
 
 
     public String[] getPropertyNames() {
```

Comment:
This method will not be called by Spring now, right?

Reply:
Yes.
Spring doesn't know we have this method.
Now this method is just used in test case.
I think it's OK to keep it, maybe we need it in future.